spacy/tests/doc/test_span.py

```diff
@@ -286,7 +296,7 @@ def test_span_attrs_writable(doc):
 
 
 def test_span_ents_property(doc):
-    """Test span.ents for the """
+    """Test span.ents for the"""
```

Comment:
What is this comment even supposed to say? "Test span.ents for the doc"?

Reply:
That's actually a good question. It looks like the comment has been like that since 2018, I just modified it due to running black on the code. 

I guess it can just be changed to `Test span.ents` or removed?

Reply:
I know you didn't write it, it just jumped out at me when I was reading through.